src/pvrclient-nextpvr.h

```diff
@@ -89,6 +89,7 @@ class cPVRClientNextPVR : P8PLATFORM::CThread
   bool CanSeekStream(void);
   bool IsTimeshifting(void);
   bool IsRealTimeStream(void);
+  bool IsRadio() {return m_nowPlaying == Radio;};
```

Comment:
Space before return and before closing bracket for clarity.

Like this: `{ return m_nowPlaying == Radio; };`

Reply:
Thanks for the tip.  Changed.   I also got to practice amending an earlier commit using the method you gave me on another PR, well worth knowing.